sphinx/application/network/concepts/queries/concept-network-query-save-profile/src/main/java/chat/sphinx/concept_network_query_save_profile/NetworkQuerySaveProfile.kt

```diff
@@ -0,0 +1,22 @@
+package chat.sphinx.concept_network_query_save_profile
+
+import chat.sphinx.concept_network_query_save_profile.model.PersonInfoDto
+import chat.sphinx.concept_network_query_save_profile.model.SaveProfileDto
+import chat.sphinx.kotlin_response.LoadResponse
+import chat.sphinx.kotlin_response.ResponseError
+import chat.sphinx.wrapper_relay.AuthorizationToken
+import chat.sphinx.wrapper_relay.RelayUrl
+import kotlinx.coroutines.flow.Flow
+
+abstract class NetworkQuerySaveProfile {
+
+    abstract fun getProfileByKey(
```

Comment:
Rename to getPeopleProfileByKey to be more descriptive

Reply:
resolved in a74452a